addon/libconiks.so

```diff
@@ -0,0 +1 @@
+../chrome/content/libconiks.so
```

Comment:
Please remove this symlink and put the `.so` in `addon/` since it's specific to the testing environment.

Another option might be to have the `.travis.yml` build it on demand, which'll keep it up to date.